test/test_service.rb

```diff
@@ -40,7 +40,7 @@ def test_construct_our_own_service
     assert_equal(created.spec.ports.size, our_service.spec.ports.size)
 
     # Check that original entity_config is not modified by kind/apiVersion patches:
-    assert_equal(our_service.kind, nil)
+    assert_nil our_service.kind
```

Comment:
I've ran rubocop and it said that `assert_nil our_service.kind` is totally fine. Perhaps we need to update the rubocop config?